_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -596,50 +594,43 @@ def get_geo_values(self, geo_type):
             self.geo_lists[geo_type] = set(crosswalk[geo_type])
             return self.geo_lists[geo_type]
 
-    def get_geos_within(self, container_geocode, contained_geocode_type,container_geocode_type):
+    def get_geos_within(self, container_geocode, contained_geocode_type, container_geocode_type):
         """
-        Return all contained regions within same container geocode.
+        Return all contained regions of the given type within the given container geocode.
 
-        Given a container geocode (e.g. "ca" for California, a state)
-        and an enclosed contained geocode type (e.g. "county").
-        return a set of container geocode value of the specified type that
-        lie within the specified geo code (e.g. all counties within California)
-        Support these 3 types:
+        Given a container_geocode (e.g "ca" for California) of a container_geocode_type
+        (e.g "state"), return:
+            - all (contained_geocode_type) within that (container_geocode_type)
+
+        Supports these 3 combinations:
             - all states within a nation
             - all counties within a state
             - all states within an hhs region
 
         Parameters
         ----------
-        container_geocode: str of identity of nation/state/hhs
-            "fips" for return container_geocode of state
-            "state_id" for return container_geocode of nation and hhs
+        container_geocode: str
+            Instance of nation/state/hhs to find the sub-regions of
         contained_geocode_type: str
-            One of "state","county"
+            The subregion type to get all instance of. One of "state", "county"
         container_geocode_type: str
-            One of "state","nation","hhs"
+            The parent region type. One of "state", "nation", "hhs"
 
         Returns
         -------
-        Set of geo ids of the specified type that lie within the specified container geocode,
-        all in string format.
+        Set of geo code strings of the given type that lie within the given container geocode.
         """
-        if contained_geocode_type not in ("county","state"):
-            raise ValueError("contained_geocode_type must be one of state, nation and hhs")
```

Comment:
so i'm generally in favor of putting quick test+exit/raise behavior before slow+process behavior to reduce cognitive load while reading, which would mean rejecting this particular change. in this case i see the benefit of a single exit condition based on more complex logic, but just to keep in mind for future prs